lib/rules/rule-nesting-at-rule-required-list/index.mjs

```diff
@@ -0,0 +1,80 @@
+import { isAtRule, isRoot } from '../../utils/typeGuards.mjs';
+import { isRegExp, isString } from '../../utils/validateTypes.mjs';
+import isStandardSyntaxRule from '../../utils/isStandardSyntaxRule.mjs';
+import matchesStringOrRegExp from '../../utils/matchesStringOrRegExp.mjs';
+import report from '../../utils/report.mjs';
+import ruleMessages from '../../utils/ruleMessages.mjs';
+import validateOptions from '../../utils/validateOptions.mjs';
+
+const ruleName = 'rule-nesting-at-rule-required-list';
+
+const messages = ruleMessages(ruleName, {
+	expected: (atRulePatterns) =>
+		`Expected rule to be nested in an at-rule matching pattern(s) "${atRulePatterns}"`,
```

Comment:
[suggestion] How about making the message a bit more helpful by wrapping each at-rule with quotes (")? For example:

```diff
-Expected rule to be nested in an at-rule matching pattern(s) "layer, /^scope/"
+Expected rule to be nested in an at-rule matching pattern(s): "layer", "/^scope/"
```

```suggestion
	expected: (...atRulePatterns) => {
		const patterns = atRulePatterns.map((pattern) => `"${pattern}"`).join(', ');

		return `Expected rule to be nested in an at-rule matching pattern(s): ${patterns}`;
	},
```

If doing so, users could customize the message more flexibly with an array of patterns, instead of a string as patterns.

Reply:
Nice catch.